### ISSUE TITLE
onmouseover labels for the color key are wrong on the calendar page

### ISSUE THREAD

**Issue by JB-Walker** [freeCodeCamp/freeCodeCamp#16124]
Moving left to right, the labels currently read : "less than 1 items", "between 1 and 2 items", "between 2 and 3 items", "more than 3 items"

They should read: "zero items", "1 item", "2 items", "3 or more items"

![labelswrong](https://user-images.githubusercontent.com/17031960/33246442-ff97bf86-d2d9-11e7-8f0c-5c0127731c09.jpg)

**Comment by raisedadead** [comment]
Hi @JB-Walker 

Thanks for reporting.

Do you mind confirming this is the case on the beta sire or the main site, also please provide additional info as presented in the issue template, regarding the environment.

Happy coding!

**Comment by JB-Walker** [comment]
Main site. I don't know whether it also occurs on beta (no account there).

Environment? Hopefully this is what you asked for:
- https://www.freecodecamp.org/jb-walker
- Chrome Version 62.0.3202.94 (Official Build) (64-bit)
- Dell Latitude E6420 Laptop
- Windows 10 Pro Version 1703
- OS Build 15063.726

**Comment by raisedadead** [comment]
@JB-Walker Perfect, that's what we needed. Thanks a lot.

**Comment by raisedadead** [comment]
Can anyone reproduce this, preferably on a windows machine?

**Comment by scissorsneedfoodtoo** [comment]
Sure, I can reproduce the issue on my Windows machine here at work.

- https://www.freecodecamp.org/scissorsneedfoodtoo
- Windows 7
- Chrome Version 62.0.3202.94 (Official Build) (32-bit)

I looked into it a bit and it seems that the issue has to do with how legends in the Cal-HeatMap module works (documentation [here](http://cal-heatmap.com/#legend)). It takes an array of N values as arguments and produces the legend of colors in question. Unfortunately, it seems like the default behavior (more [here](http://cal-heatmap.com/#legendTitleFormat)) is to produce an svg of the legend rectangles within the larger heat map calendar svg with the mouseover text as titles. 

It's not ideal, but I've tried to change the title content of the legend rectangles with css, but was unsuccessful. There's probably a much better way to change the legend's title text. I'll play around with the page's JS a little later.
